### ISSUE TITLE
[Blog] 2018: Interlinking - links to mclabs.com

### ISSUE THREAD

**Issue by esuger** [MicroTekAppDev/Attune.Website.Issues#17]
**Page Info**
https://www.attune.com/insights/blog/posts/2018/january/what-is-a-virtual-training-lab/

**Describe the bug**
www.mclabs.com links on this page.  " virtual training labs" and " hybrid learning" link to mclabs.com

**Screenshots**

**Additional context**

**Comment by esuger** [comment]
"hybrid learning" link goes to www.mlcabs.com on this page:

https://www.attune.com/insights/blog/posts/2018/january/what-is-a-virtual-training-lab/

**Comment by esuger** [comment]
"hybrid learning" link goes to info.mlcabs.com on this page:

https://www.attune.com/insights/blog/posts/2018/january/training-room-essentials/

**Comment by esuger** [comment]
https://www.attune.com/insights/blog/posts/2018/february/marketing-your-training-program/
--

**Comment by esuger** [comment]
https://www.attune.com/insights/blog/posts/2018/february/value-of-using-managed-services-for-training/

**Comment by esuger** [comment]
Many of the blog posts have bad links to www.mclabs.com,  blog.mclabs.com, or info.mclabs.com.    I am going to wait on any update before reviewing any further.

**Comment by aj-mclabs** [comment]
Duplicate of #14

**Comment by aj-mclabs** [comment]
Thread locked pending interlink fixes.  Will notify when 2018 Blogs can be rechecked.

**Comment by aj-mclabs** [comment]
2018 interlinks have been fixed so if you find any mclabs links post them below.

**Comment by esuger** [comment]
Still a bad link for "FREE EBOOK: Complete Training Program Planning Guide".

https://test-umb.attune.com/insights/blog/posts/2018/november/4-key-steps-in-a-successful-event-project-implementation-plan/

**Comment by esuger** [comment]
Link "RELATED ARTICLE: What is hybrid learning and how does it differ from other virtual training strategies?" on page https://test-umb.attune.com/insights/blog/posts/2018/august/when-to-outsource-training-with-a-managed-services-partner/ points to www.mclabs.com

**Comment by esuger** [comment]
On this page: https://test-umb.attune.com/insights/blog/posts/2018/june/when-cloud-based-learning-offers-the-ideal-virtual-training-environment/  the " cloud learning tools" link points to www.mclabs.com

**Comment by esuger** [comment]
On this page: https://test-umb.attune.com/insights/blog/posts/2018/february/5-ways-hybrid-learning-enhances-instructor-led-training/ the " variety of options for remote guest speakers/lecturers" link points to www.mclabs.com

**Comment by esuger** [comment]
On this page: https://test-umb.attune.com/insights/blog/posts/2018/january/training-room-essentials/ the "RELATED ARTICLE: Increase the retention level of your learners by using these techniques to engage and focus your audience." link points to www.mclabs.com

**Comment by aj-mclabs** [comment]
These have been fixed.
